### ISSUE TITLE
options.whitelist overwrites merged operators and whitelist 

### ISSUE THREAD

**Issue by DaddyWarbucks** [feathersjs-ecosystem/feathers-sequelize#292]
**Edit** I realized I did not describe the actual bug very well, only the solution. If you do not provide a `whitelist` property, then the adapter works as expected and the `service.options.whitelist` includes all of the default operators like `['$like', '$iLike', ...]`. But, if you provide a whitelist, then `service.options.whitelist` is _only_ the provided whitelist and not the merged array. This `whitelist` is actually turned over to the commons adapter as the `operators` array. This means that if you provide a whitelist, that it removes all default operators and _only_ your operators are used.

On This line: https://github.com/feathersjs-ecosystem/feathers-sequelize/blob/5c66df1e460b15255fa118cbb11f6834a889fbbd/lib/index.js#L38

The sequelize adapter service creates a new `whitelist` variable that is the merged operator names and user supplied whitelist array, creating the full set of allowed query operators.

But, in this line: https://github.com/feathersjs-ecosystem/feathers-sequelize/blob/5c66df1e460b15255fa118cbb11f6834a889fbbd/lib/index.js#L46

The value is overwritten by `options.whitelist` as `Object.assign()` merges right to left. I am not sure of the best solution here. Simply flipping the order of the object and `options` would mean `id` would also be overwritten making options less "true". `operators` is also being overwritten. I did notice that `operators` is not used in common AdapterService here: https://github.com/feathersjs/feathers/blob/b325045700d94db65c26e691547d2542d0779098/packages/adapter-commons/lib/service.js#L42
 But, I thought it may be used for other things or was maybe passed in prep for an upcoming change/feature.

I think the easiest solution for this would be

```
    super(Object.assign({
      id,     
    }, options, { whitelist, operators }));
```

This also raises a question to me of how would I exclude a default operator if I didn't want to use it? I don't currently have a need for this, but I think it would be easy enough to create a `blacklist` prop that would remove any unwanted default operators. Maybe something like

```
const whitelist = alreadyMergedWhitelistAndOperators.filter(operator => options.blacklist.includes(operator))
```

**Comment by DaddyWarbucks** [comment]
The usage of `this.options.operators` is currently not using the merged whitelist list either
https://github.com/feathersjs-ecosystem/feathers-sequelize/blob/5c66df1e460b15255fa118cbb11f6834a889fbbd/lib/index.js#L80

**Comment by DaddyWarbucks** [comment]
Please see attached screenshots:

When no whitelist is provided, the `service.options.whitelist` is the `keys` from `operators`
<img width="1920" alt="Screen Shot 2019-04-18 at 1 34 43 PM (2)" src="https://user-images.githubusercontent.com/4671808/56383169-d96dda80-61de-11e9-8c0d-335e7c887ce5.png">

When a whitelist is provided, `service.options.whitelist` is _just_ that list and not the merged keys from operators
<img width="1920" alt="Screen Shot 2019-04-18 at 1 34 54 PM (2)" src="https://user-images.githubusercontent.com/4671808/56383209-f2768b80-61de-11e9-886e-4be3b40219c8.png">

**Comment by daffl** [comment]
Closed via #293
